test/Microsoft.Azure.Devices.Edge.Test.Common/linux/EdgeDaemon.cs

```diff
@@ -188,29 +184,21 @@ public async Task UninstallAsync(CancellationToken token)
 
             string[] commands = this.packageManagement.GetUninstallCommands();
 
-            foreach (string command in commands)
-            {
-                try
+            await Profiler.Run(
+                async () =>
                 {
-                    await Profiler.Run(
-                    async () =>
+                    foreach (string command in commands)
                     {
-                        string[] output = await Process.RunAsync("bash", $"-c \"{string.Join(" || exit $?; ", command)}\"", token);
-                        if (output.Length > 0)
+                        try
                         {
-                            Log.Verbose($"Uninstall command '{command}' ran unsuccessfully. This is probably because this component wasn't installed. Output:\n" + string.Join("\n", output));
+                            await Process.RunAsync("bash", $"-c \"{string.Join(" || exit $?; ", command)}\"", token);
                         }
-                        else
+                        catch (Win32Exception e)
```

Comment:
I always thought it was weird to see this exception type on linux. I'm assuming the name makes sense with context, but think I am missing some.

Reply:
I know right? On forums like SO I've noticed people complaining. But yes, this is a real thing for .NET Core on Linux. I'm sure it's just a historical artifact since .NET came from Windows...